xtend-contrib/src/main/java/de/oehme/xtend/contrib/Builder.xtend

```diff
@@ -54,6 +67,15 @@ class BuilderProcessor extends AbstractClassProcessor {
 					'''
 					primarySourceElement = cls
 				]
+				addMethod('''set«builtField.simpleName.toFirstUpper»''') [
```

Comment:
Ah, I didn't think about that. Then we don't need the 'build' and 'copy' methods at all, all clients can use 'builder' and 'copier', in Xtend having to use '=>'.